docs/sql-ref-syntax-qry-select-transform.md

```diff
@@ -57,16 +65,38 @@ SELECT TRANSFORM ( expression [ , ... ] )
 
     Specifies a command or a path to script to process data.
 
-### SerDe behavior
+### ROW FORMAT DELIMITED BEHAVIOR
+
+When spark use `ROW FORMAT DELIMITED` format, Spark will use `\u0001` as default filed delimit,
+use `\n` as default line delimit and use `"\N"` as `NULL` value in order to differentiate `NULL` values 
+from empty strings. These delimit can be overridden by `FIELDS TERMINATED BY`, `LINES TERMINATED BY` and
+`NULL TERMINATED AS`. Since we use `to_json` and `from_json` to handle complex data type, so 
```

Comment:
> Since we use `to_json` and `from_json` to handle complex data type, so 
`COLLECTION ITEMS TERMINATED BY` and `MAP KEYS TERMINATED BY` won't work in current code.

I can't follow what this is saying ..

Reply:
I am I know we use to_json and from_json internally. but how does it relate to `COLLECTION ITEMS TERMINATED BY` and `MAP KEYS TERMINATED BY`, and why they don't work?

Reply:
How about current